releasenotes/notes/removed_deprecation_0.20-17b0ea6a3980eb4f.yaml

```diff
@@ -0,0 +1,26 @@
+---
+upgrade:
+  - |
+    Code and tests deprecated in 0.20 are removed.
+    :class:`.visualization.QCircuitImage` objects no longer supports
+    `qregs`, `cregs`, `layout`and `global_phase` arguments.
+    
+  - |
+    Code and tests deprecated in 0.20 are removed. In
+    :class:`.transpiler.CouplingMap` :meth:`~subgraph` is removed.
+    :meth:`~reduce` can be used in place of 
+    :meth:`~subgraph`.
+    
+  - |
+    Code and tests deprecated in 0.20 are removed. In
+    :class:`.dagcircuit.DAGCircuit` private :meth:`~_copy_circuit_metadata` is removed.
+    Instead use public :meth:`~copy_empty_like` with same functionality.
+  
+  - |
+    Code and tests deprecated in 0.20 are removed. In
+    :class:`.synthesis.SuzukiTrotter` deprecation warning for
+    odd `order` is replaced with `ValueError`.
+    Refer to `#10747 <https://github.com/Qiskit/qiskit/issues/10747>`
```

Comment:
```suggestion
    The parameter `order` in :class:`.synthesis.SuzukiTrotter` raises an exception instead of deprecation warning when set in an odd number. Suzuki product formulae are symmetric and therefore only defined for even orders.
```

Reply:
Hmm, your suggestions are quite good, i have tried to accommodate them in the commit. Thanks